docs/en/observability/synthetics-reference/lightweight-config/common.asciidoc

```diff
@@ -234,6 +234,78 @@ By default, monitors are automatically retested if the monitor goes from "up" to
 retest_on_failure: false
 ----
 
+////////////////////////
+locations
```

Comment:
General comment that you really can't fix: It's annoying that the user needs to use a scrollbar to view all the content in the table.